### ISSUE TITLE
Message#Unknown: 操作过于频繁，可稍后再试。

### ISSUE THREAD

**Issue by huan** [padlocal/wechaty-puppet-padlocal#109]
When Friday BOT has sent too many UrlLinks, the server will response with `操作过于频繁，可稍后再试。`

```
22:43:53 VERB IoClient onMessage(Message#Unknown[🗣Contact<Friday BOT>@👥Room<ChatOps - Friday BOT>]     操作过于频繁，可稍后再试。)
```

However, this message type is set as `Unknown`, I believe it should be set to a specific type then we will be able to deal with it.

**Comment by padlocal** [comment]
@huan Good catch! Could you please provide the raw message payload if happens again.

**Comment by huan** [comment]
Ok, will do it!
